presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -75,6 +75,7 @@
     EXCEEDED_TIME_LIMIT(0x0002_0003),
     CLUSTER_OUT_OF_MEMORY(0x0002_0004),
     EXCEEDED_CPU_LIMIT(0x0002_0005),
+    INVALID_PREPARED_STATEMENT(0x0002_0006),
```

Comment:
This is the `INSUFFICIENT_RESOURCES` error code group. This should go under `USER_ERROR`. However, we don't need a new error code -- `NOT_SUPPORTED` is fine here.